docs: Account for removal of dashboard app generation (#6496)
cube-js/cube: docs/content/Workspace/Developer-Playground.mdx

  - /cloud/dev-tools/dev-playground
---

> Playground is a web-based tool which allows to execute queries and preview the results.

Reviewer: ```suggestion
Playground is a web-based tool which allows executing queries and previewing their results.
```
